tools/add_strelka2_fields.cwl

```diff
@@ -0,0 +1,45 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: add_strelka2_fields
+doc: >-
+  This tool will run a Python script that takes a Strelka2 tumor-normal VCF and adds
+      canonical fields like GT and AD
+requirements:
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/add-strelka2-fields:1.0.0'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 3000
+    coresMin: 4
+  - class: ShellCommandRequirement
+
+baseCommand: [/usr/bin/add_strelka2_fields.py]
+
+inputs:
+  strelka2_vcf:
```

Comment:
To avoid the "No index found warning" please add a `secondaryFiles` requirement to this for `.tbi`. Even though it technically works without out it, I'd like it to run clean. Also in the PR please add yourself ass the Assignee and Labels. Lastly, once the changes are made and tested locally, you can also add a test run on cavatica....unless you don't have rabix composer still.